HW0/P4/P4.py

```diff
@@ -55,8 +68,28 @@
     # Initial conditions for s0
     # Compute the rest of sk using Eq (1)
 
-    # ax.plot(x_coords, y_coords, z_coords,
-    #         '-k', label='Blind trajectory')
+    A = np.matrix([[1, 0, 0, dt, 0, 0],
```

Comment:
Like the previous comment, consider using np.diag here. You can also use hstack or vstack to combine several diagonal matricies.